lighthouse-extension/app/styles/lighthouse.css

```diff
@@ -124,9 +137,23 @@ body {
   box-shadow: 0px 2px 4px 0px rgba(0,0,0,0.50);
   border-radius: 2px;
   padding: 8px 16px 10px 16px;
+  margin-left: auto;
+  align-self: stretched;
 }
 
-.status {
+.button--configure {
```

Comment:
Let's use buttons for "options" and "back" that are similar to "generate report" but smaller and a different color.
Slightly prefer the feel of buttons vs these links.